DTSPB-4521 Remove HMRC PP Journey

MarcusHMCTS opened the pull request:
### JIRA link (if applicable) ###

[DTSPB-4521](https://tools.hmcts.net/jira/browse/DTSPB-4521)--Amendments to PP Journey before Deceased Died 01 Jan 22

[DTSPB-4527](https://tools.hmcts.net/jira/browse/DTSPB-4527)--Amendments to PP Journey after Deceased Died 01 Jan 22
### Change description ###



**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

MarcusHMCTS committed:
DTSPB-4521 Remove HMRC PP Journey

MarcusHMCTS committed:
Merge branch 'master' into DTSPB-4521_Remove_HMRC_PP_Journey

MarcusHMCTS committed:
DTSPB-4521 add dod before 2022 IHTFormID validation

MarcusHMCTS committed:
DTSPB-4521 add dod before 2022 IHTFormID validation

MarcusHMCTS committed:
DTSPB-4521 add dod before 2022 IHTFormID validation Unit Test

MarcusHMCTS committed:
DTSPB-4527 add dod after 2022 changes

MarcusHMCTS committed:
DTSPB-4521 update the IHTFormIDValidationRule with the DOD

MarcusHMCTS committed:
DTSPB-4527 default IhtFormEstate to IHT400

MarcusHMCTS committed:
DTSPB-4527 add default IhtFormEstate to IHT400 unit test

MarcusHMCTS committed:
DTSPB-4527 fix missing text

MarcusHMCTS committed:
Merge branch 'master' into DTSPB-4521_Remove_HMRC_PP_Journey

FeliTam commented on ccdImports/configFiles/CCD_Probate_Backoffice/CaseEventToFields.json:
```diff
@@ -3180,8 +3180,8 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "applyforGrantPaperApplicationMan", "CaseFieldID": "foreignAssetEstateValue", "CaseEventFieldLabel": "Specify how much", "PageFieldDisplayOrder": 12, "DisplayContext": "OPTIONAL", "PageID": "applyforGrantPaperApplicationManPage2A", "PageDisplayOrder": 2, "FieldShowCondition": "dateOfDeathAfterEstateSwitch=\"No\" AN ...
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "applyforGrantPaperApplicationMan", "CaseFieldID": "hmrcLetterId", "PageFieldDisplayOrder": 1, "DisplayContext": "OPTIONAL", "PageID": "applyforGrantPaperApplicationManPage2B", "PageDisplayOrder": 3, "PageLabel": "HMRC letter", "PageShowCondition": "(dateOfDeathAfterEstateSwitch=\"No\" AND ihtFormId=\"IHT400\") OR  ...
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "applyforGrantPaperApplicationMan", "CaseFieldID": "uniqueProbateCodeId", "PageFieldDisplayOrder": 2, "DisplayContext": "OPTIONAL", "PageID": "applyforGrantPaperApplicationManPage2B", "PageDisplayOrder": 3, "PageLabel": "HMRC letter", "FieldShowCondition": "hmrcLetterId=\"Yes\"", "ShowSummaryChangeOption": "y", "Ca ...
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "applyforGrantPaperApplicationMan", "CaseFieldID": "hmrcLetterId", "PageFieldDisplayOrder": 1, "DisplayContext": "OPTIONAL", "PageID": "applyforGrantPaperApplicationManPage2B", "PageDisplayOrder": 3, "PageLabel": "HMRR code", "PageShowCondition": "(dateOfDeathAfterEstateSwitch=\"No\" AND ihtFormId=\"IHT400\") OR (d ...
```
typo here, should be HMRC

MarcusHMCTS committed:
DTSPB-4517 fix typo

MarcusHMCTS committed:
Merge branch 'master' into DTSPB-4521_Remove_HMRC_PP_Journey

MarcusHMCTS committed:
DTSPB-4517 fix default 400 with ihtFormEstateValuesCompleted=Yes

tom-saunders-cts committed:
add ccd config as an input to buildAllXlsx.

This means the xslx files correctly rebuild when they are modified.

tom-saunders-cts committed:
Update ccd defs for died after path.

Some of these changes will likely bleed into other pages, so will need
to check those aren't causing additional problems.

tom-saunders-cts committed:
Change estate to assets for probate for net value label.

This was run past the service team in standup.

tom-saunders-cts committed:
increase devmemoryLimits?

tom-saunders-cts committed:
Update after review feedback.

tom-saunders-cts committed:
DTSPB-4570 changes for caseworker manual creation IHT (#3241)

DTSPB-4579 caseworker iht changes

tom-saunders-cts committed:
Add error handling for selecting 400&421 form.

tom-saunders-cts committed:
Updated test coverage for IHTForm validation.

This now validates two different fields which can have slightly different values.

tom-saunders-cts committed:
checkstyle issue.

tom-saunders-cts committed:
DTSPB-4579 report error when selecting both IHT400&421 for caseworker create (#3255)

DTSPB-4579 report error on selecting IHT400&421 for caseworker case creation

MarcusHMCTS committed:
DTSPB-4521 fix ihtFormEstate null issue

MarcusHMCTS committed:
Merge branch 'master' into DTSPB-4521_Remove_HMRC_PP_Journey

MarcusHMCTS committed:
Merge branch 'master' into DTSPB-4521_Remove_HMRC_PP_Journey

MarcusHMCTS committed:
DTSPB-4521 update deceasedHadLateSpouseOrCivilPartner label

MarcusHMCTS committed:
DTSPB-4521 update ihtNetValue CaseEventFieldLabel

MarcusHMCTS committed:
DTSPB-4521 update ihtFormEstateValuesCompleted CaseEventFieldLabel

MarcusHMCTS committed:
DTSPB-4521 fix label with late husband, wife

MarcusHMCTS committed:
Merge branch 'master' into DTSPB-4521_Remove_HMRC_PP_Journey

MarianIrudayanathan committed:
Dtspb 4521 fix e2e tests (#3283)

* Update dependency commons-io:commons-io to v2.20.0 (#3263)

* Update dependency commons-io:commons-io to v2.20.0

* fix load-local-environment-variables.sh

---------

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: FeliTam <feli.tam@justice.gov.uk>

* Update Node.js to v22.17.1 (#3264)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

* Update dependency @types/node to v22.16.5 (#3225)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

* Update dependency org.springframework.security:spring-security-test to v6.5.2 (#3265)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

* Update plugin com.github.hmcts.rse-cft-lib to v0.19.1810 (#3266)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

* DTSPB-4895 Fix Handoff Reason issue (#3250)

* DTSPB-4895 Fix Handoff Reason issue

* DTSPB-4895 Fix instatiate issue

* DTSPB-4895 Added tests

* DTSPB-4895 removed unnecessary file

* Update dependency webdriverio to v9.18.4 (#3227)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

* Update dependency eslint-plugin-mocha to v11 (#3267)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

* DTSPB-4895 fix setHandoffReason (#3274)

* DTSPB-4912 reset caveat history correction events (#3275)

* Update spring boot to v3.5.4 (#3278)

* Update spring boot to v3.5.4

* update spring cloud

* resolve commons-fileupload

* update comment

---------

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: FeliTam <feli.tam@justice.gov.uk>

* DTSPB-4914 fix caveat notification (#3277)

* DTSPB-4914 bump commons version

* update log

* Add suppression for Apache Commons Lang CVE.

We do not directly use this functionality.

* Update dependency org.pitest:pitest to v1.20.1 (#3279)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

* DTSPB-4521: Update the e2e tests for the new IHT changes

* DTSPB-4521: Revert the changes to config.js

---------

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: FeliTam <feli.tam@justice.gov.uk>
Co-authored-by: Iswarya Pepakayala <Iswarya.Pepakayala@justice.gov.uk>
Co-authored-by: Tom Saunders <tom.saunders@justice.gov.uk>
Co-authored-by: MarcusHMCTS <Marcus.Ho@justice.gov.uk>

MarcusHMCTS committed:
Merge branch 'master' into DTSPB-4521_Remove_HMRC_PP_Journey

MarcusHMCTS committed:
DTSPB-4521 add back ihtFormEstate option

MarcusHMCTS committed:
DTSPB-4521 fix e2e issue

MarcusHMCTS committed:
DTSPB-4521 fix e2e issue

tom-saunders-cts reviewed (commented):
looking only at changes [here](https://github.com/hmcts/probate-back-office/pull/3214/files/0d0454128a601fed5244c2ed9840b1d358c2af96..46b44d4b6ec07191ad4425873bb833796c62639a) this seems reasonable to re-add the options for the iht forms, but i've not gone into anything beyond that

MarcusHMCTS committed:
Merge branch 'master' into DTSPB-4521_Remove_HMRC_PP_Journey

MarcusHMCTS committed:
Merge branch 'master' into DTSPB-4521_Remove_HMRC_PP_Journey
